open-sphere-base/control-panels/src/main/java/io/opensphere/controlpanels/toolbox/ControlPanelToolboxImpl.java

```diff
@@ -86,38 +72,37 @@ protected DetailPanelProviderRegistry createDetailPanelProviderRegistry(Toolbox
     public String getDescription()
     {
         return "A toolbox extension for control panels";
+
     }
 
-    /**
-     * {@inheritDoc}
-     *
-     * @see io.opensphere.controlpanels.ControlPanelToolbox#getDetailPanelProviderRegistry()
-     */
     @Override
-    public DetailPanelProviderRegistry getDetailPanelProviderRegistry()
+    public DetailPanelProvider getDefaultDetailPanelProvider()
     {
-        return myDetailPanelProviderRegistry;
+        return myDefaultDetailPanelProvider;
     }
 
-    /**
-     * {@inheritDoc}
-     *
-     * @see io.opensphere.controlpanels.ControlPanelToolbox#getUISpan()
-     */
     @Override
     public ObservableValue<TimeSpan> getUISpan()
     {
         return myUISpan;
     }
 
-    /**
-     * {@inheritDoc}
-     *
-     * @see io.opensphere.controlpanels.ControlPanelToolbox#setUISpan(ObservableValue) ()
-     */
     @Override
     public void setUISpan(ObservableValue<TimeSpan> uiSpan)
     {
         this.myUISpan = uiSpan;
+
```

Comment:
Remove the extra blank line (you can get Eclipse to remove extra lines for you on save)